repos.yaml

```diff
@@ -631,6 +631,20 @@
       url: 'https://pypi.python.org/pypi/{name}/'
   tags: [ all, preproduction, fastfetch ]
 
+- name: rubygem
+  desc: RubyGems
+  family: rubygem
+  sources:
+    - name: latest_specs.4.8
+      fetcher: File
+      parser: RubyGem
+      url: https://api.rubygems.org/latest_specs.4.8.gz
+      compression: gz
+  links:
+    - desc: RubyGem page
+      url: 'https://www.rubygems.org/gems/{name}/'
+  tags: [ all, have_testdata, demo, production, fastfetch ]
```

Comment:
This doesn't have testdata (would be nice to add some btw). And I'd not add it to production just yet.